geopandas/plotting.py

```diff
@@ -102,9 +106,8 @@ def plot_series(s, colormap='Set1', axes=None, **color_kwds):
     """
     import matplotlib.pyplot as plt
     if axes is None:
-        fig = plt.gcf()
-        fig.add_subplot(111, aspect='equal')
-        ax = plt.gca()
+        fig, ax = plt.subplots(figsize=figsize)
```

Comment:
Well spotted. I intentionally changed it to this new behaviour because one might argue that if the user wants to reuse a pre-existing figure then she should explicitly pass the desired Axes as an argument. It is true that with the old code an existing figure would be re-used, but it would actually create a new Axes. So either (i) the existing figure is empty, in which case the old and new behaviour are equivalent; (ii) the figure contains a single subplot with some contents, in which case that content is overwritten by adding an extra Axes on top of the old one (most likely undesired behaviour; see example below), or (iii) the figure contains multiple subplots, in which case the old code would add an extra Axes on top of all the existing ones (certainly undesired behaviour).

Here is a simple example of where the old behaviour fails:
```
import matplotlib.pyplot as plt

# Create a "pre-existing" figure
fig1 = plt.figure()
ax1 = fig1.gca()
ax1.plot([1, 2, 3], [4, 3, 5])

# This is what the old code would have done:
# a new Axes is created on top of the old one
fig = plt.gcf()
fig.add_subplot(111, aspect='equal')
ax = plt.gca()
ax.plot([3, 5, 7], [2, 3, 4])
```